source/development/feature-flags.html.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Feature flags in GOV.UK Frontend
+weight: 1
+last_reviewed_on: 2023-11-28
+review_in: 6 months
+---
+# Using feature flags in GOV.UK Frontend
+
+Feature flags are a way for us to release large scale or experimental work early by allowing users to opt in our out of a new, breaking change or feature. Use this guidance to decide if a feature flag is useful for your use case.
+
+## When a feature flag is useful
+Feature flags are useful when the following conditions are true.
```

Comment:
It could be because I'm reviewing in this layout, but this sentence looks out of place on its own. Is it worth having the subheads in a bullet list here as well? As a kind of summary?

Reply:
I think that's a good idea. I've implemented this, let me know what you think.